components/PartForm/index.js

```diff
@@ -15,7 +16,10 @@ export default function PartForm({ onSubmit, formName, defaultData }) {
     const year = allDate.getFullYear();
     const todayDate = day + "." + month + "." + year;
 
+    const uuid4 = uuidv4();
+
     const newPart = {
+      uuid: uuid4,
```

Comment:
You can just use `uuid4()` while creating the object `newPart` and save one line of code
```suggestion
    const newPart = {
      uuid: uuidv4(),
```

Reply:
oh great advice 👌 thanks
i'll fix this